sos/report/plugins/origin.py

```diff
@@ -145,32 +141,26 @@ def setup(self):
             ]
 
             self.add_cmd_output([
-                '%s get -o json %s' % (oc_cmd_admin, jcmd) for jcmd in jcmds
+                '%s get -o yaml %s' % (oc_cmd_admin, jcmd) for jcmd in jcmds
```

Comment:
Is there anything currently using the json outputs? Anything in Insights or other tooling?


Also, the `postproc()` method will need to be updated to account for not only the file name change but also any corrections to ensure it keeps working.

Reply:
Hey, I will put it back to `json`. I put it as `yaml`, but it should be `json`.